noobaa/noobaa-operator
# Integrate with KEDA for custom metrics scaling

### Explain the changes
1. Integrate with KEDA for custom metrics scaling: Initial changes
     This PR will have only Keda related changes, not prometheus-adaptor 

Signed-off-by: Naveen Paul @naveenpaul1 

### Issues: Fixed #xxx / Gap #xxx
1. https://issues.redhat.com/browse/MCGI-101

### Testing Instructions:
1. Install noobaa in a cluster after installing keda and Prometheus
2. check for the newly created Keda HPA
